pkg/framework/aries/framework_test.go

```diff
@@ -16,6 +16,8 @@ import (
 	"strings"
 	"testing"
 
+	"github.com/hyperledger/aries-framework-go/pkg/storage"
+
```

Comment:
should be grouped with the other framework imports.

Reply:
fixed